hangups/hangouts.proto

```diff
@@ -391,17 +390,17 @@ message Conversation {
   optional UserConversationState self_conversation_state = 4;
   repeated UserReadState read_state = 8;
 
-  // unknown (0) = 9;
+  // "has_active_hangout" (0) = 9;
 
   optional OffTheRecordStatus otr_status = 10;
 
-  // unknown (1) = 11;
+  // "otr_toggle" (1) = 11;
 
   repeated ParticipantId current_participant = 13;
   repeated ConversationParticipantData participant_data = 14;
-
-  // unknown ([1]) = 18;
-  // unknown (0) = 19;
+  // "fork_on_external_invite" = 17
+  // "network_type" ([1]) = 18;
+  // "force_history_state" (0) = 19;
```

Comment:
Ah, I'm guessing you noticed that the order of the keys in the takeout data matches the order of the protobuf fields.